client/profile-wizard/steps/industry.js

```diff
@@ -97,16 +107,15 @@ class Industry extends Component {
 		await updateProfileItems( { industry: this.state.selected } );
 
 		if ( ! isError ) {
-			goToNextStep();
-		} else {
-			createNotice(
-				'error',
-				__(
-					'There was a problem updating your industries',
-					'woocommerce-admin'
-				)
-			);
+			return true;
```

Comment:
I had a look around but I couldn't determine how this return value is used, and in the other flow  there's no return value. 

There's a .onContinue().then() below  - which would mean that it continues with the .then(goToNextStep()) even when isError == true

Could this be an unintended change?